<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_select</class>
 <widget class="QDialog" name="Dialog_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="sel_in">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入要查询的学生学号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sel_buto">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确认输入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ret">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>470</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>160</width>
     <height>293</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="id"/>
    </item>
    <item>
     <widget class="QLineEdit" name="name"/>
    </item>
    <item>
     <widget class="QLineEdit" name="sex"/>
    </item>
    <item>
     <widget class="QLineEdit" name="phoneNum"/>
    </item>
    <item>
     <widget class="QLineEdit" name="birth"/>
    </item>
    <item>
     <widget class="QLineEdit" name="idNum"/>
    </item>
    <item>
     <widget class="QLineEdit" name="nation"/>
    </item>
    <item>
     <widget class="QLineEdit" name="oriplace"/>
    </item>
    <item>
     <widget class="QLineEdit" name="remarks"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>160</width>
     <height>290</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>学号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>联系方式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>出生日期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>身份证号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>民族</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>籍贯</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>备注</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
